src/DotVVM.Framework/Compilation/Javascript/JavascriptCompilationHelper.cs

```diff
@@ -10,23 +10,6 @@ public static class JavascriptCompilationHelper
     {
         public static string CompileConstant(object obj) => JsonConvert.SerializeObject(obj, new StringEnumConverter());
 
-        private static readonly object indexerTargetParameter = new object();
-        private static readonly object indexerExpressionParameter = new object();
-        private static readonly ParametrizedCode indexerCode =
-            new JsIdentifierExpression("ko").Member("unwrap").Invoke(new JsSymbolicParameter(indexerTargetParameter)).Indexer(new JsSymbolicParameter(indexerExpressionParameter))
-            .FormatParametrizedScript();
-        [Obsolete]
-        public static ParametrizedCode AddIndexerToViewModel(ParametrizedCode script, object index, bool unwrap = false) =>
-            AddIndexerToViewModel(script, new JsLiteral(index), unwrap);
-        [Obsolete]
-        public static ParametrizedCode AddIndexerToViewModel(ParametrizedCode script, JsExpression indexer, bool unwrap = false)
-        {
-            return indexerCode.AssignParameters(o =>
-                o == indexerTargetParameter ? new CodeParameterAssignment(script) :
-                o == indexerExpressionParameter ? CodeParameterAssignment.FromExpression(indexer) :
-                default(CodeParameterAssignment));
-        }
```

Comment:
This one is a v2.0 obsolete, but I'd keep it removed, as it's not a very commonly used API.